Cargo.toml

```diff
@@ -1,3 +1,40 @@
+[patch.'https://github.com/paritytech/substrate']
```

Comment:
While we sure need this to run the CI, I am not sure we want to keep this in here always. From the top of my head I can't really see any actual drawbacks of having this in practice, but if feels icky. I need to ponder about blanket-applying this hear a bit more...